tasks/move-granules/tests/data/payload.json

```diff
@@ -20,20 +20,6 @@
       }
     },
     "granuleIdExtraction": "(MOD11A1\\..*)(\\.hdf|\\.cmr\\.xml|_[\\w]{1,}\\.jpg)",
```

Comment:
Is this also no longer necessary?

Reply:
Correct, I thought I removed that, but I must be thinking of my postToCmr PR.

Reply:
👍 19ce650ca